src/pages/inventory/tabs/TabsView.js

```diff
@@ -19,9 +19,9 @@ const Tabs = (props) => {
     name: tab.name,
     hasToolTip: true,
     toolTipText: tab.toolTipText,
-    count: `(${props.dashboardStats[tab.count].toLocaleString()})`,
-    display: [tab.name, props.dashboardStats[tab.count].toLocaleString()],
-    clsName: `${tab.name}`.toLowerCase().replace(' ', '_'),
+    count: tab.count != "none" ? `(${props.dashboardStats[tab.count].toLocaleString()})` : "(NA)",
```

Comment:
The linter is throwing warnings for the use of '!=' and recommends changing it to '!==' (for both lines 22 and 23)

Reply:
We can keep it like this and next PR we can fix or make change to this one. Passing this for now.